docs/root/manual/getting-started/run-signer-node.md

```diff
@@ -424,10 +515,75 @@ sudo iptables -L -v
 sudo service netfilter-persistent save
 ```
 
+:::tip
+
+Here is an example command created with the example set for `release-preprod`:
+
+```bash
+sudo iptables -A INPUT -s 192.168.1.75 -p tcp --dport 3128 -j ACCEPT
+sudo iptables -L -v
+sudo service netfilter-persistent save
+```
+
+:::
+
 ## Verify the Mithril Signer deployment
 
 :::tip
 There is a delay of `2` epochs between the registration of the signer node and its ability to generate individual signatures. This delay is further explained in the [Mithril certificate chain in depth](https://mithril.network/doc/mithril/mithril-protocol/certificates) documentation.
 
 Once this delay has passed, you should be able to observe your `PoolId` listed in some of the certificates accessible on the [`Mithril Explorer`](https://mithril.network/explorer).
 :::
+
+### Verify your Signer is registered
+
+After installing the Mithril Signer, you can verify that your node is registered by checking your Mithril Signer node logs.  
+
+First, download the script in the desired directory:
+
+```bash
+wget "https://raw.githubusercontent.com/input-output-hk/mithril/main/mithril-signer/scripts/verify_signer_registration.sh"
+```
+
+Make the script executable:
+
+```bash
+chmod +x verify_signer_registration.sh
+```
+
+Then, add the environment variable corresponding to the path of your Mithril Signer node logs file:
+```bash
+export SIGNER_LOGS_PATH=**YOUR_SIGNER_LOGS_PATH**
+```
+
+Finally, execute the script:
+```bash
+./verify_signer_registration.sh
+```
+
+### Verify your Signer generates individual signatures
```

Comment:
`### Verify your signer contributes with individual signatures`
Also in SPO onboarding guide :slightly_smiling_face: